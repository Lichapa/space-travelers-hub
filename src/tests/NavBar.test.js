import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Navbar from '../Components/NavBar';

describe('Navbar Component Test', () => {
  it('Renders NavBar', () => {
    const app = renderer.create(
      <Provider store={store}>
        <Router>
          <Navbar />
        </Router>
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
