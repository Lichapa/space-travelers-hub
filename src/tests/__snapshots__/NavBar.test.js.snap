// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component Test Renders NavBar 1`] = `
<nav>
  <div
    id="site-header"
  >
    <img
      alt="Logo"
      className="logo"
      src="logo.png"
    />
    <h2>
      Space Travelers' Hub
    </h2>
  </div>
  <ul
    className="site-links"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li
      className="mission-link"
    >
      <a
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li>
      <a
        href="/my-profile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;
